### PR TITLE
Katas A03 - Walter Benjamin Vaquera Reyes

### PR DESCRIPTION
**Pull request by waltervaquera**
Solutions of the Assigment 03 katas and their respective tests classes, applying Checkstyles and the SonarLint plugin.

### CODE REVIEW

**Comment by waltervaquera** [comment]
image
image
image
image
image

**Comment by marcos-x86** [comment]
|Overall Project|NaN% **`NaN%`**|:green_circle:|
|:-|:-|:-:|

> There is no coverage information present for the Files changed

File: build.gradle [hunk 1]
```diff
@@ -19,10 +19,3 @@ test {
     useTestNG()
     finalizedBy jacocoTestReport
 }
-
-jacocoTestReport {
```
Comment (marcos-x86) [inline build.gradle:23]: Why do you delete this?

File: src/test/java/bo/usfx/ExampleTests.java [hunk 2]
```diff
@@ -1,18 +0,0 @@
-package bo.usfx;
```
Comment (marcos-x86) [inline src/test/java/bo/usfx/ExampleTests.java:1]: Don't delete this file, please restore it.

File: src/main/java/bo/usfx/Example.java [hunk 3]
```diff
@@ -1,14 +0,0 @@
-package bo.usfx;
```
Comment (marcos-x86) [inline src/main/java/bo/usfx/Example.java:1]: Don't delete this file, please restore it.

File: coding
Comment (marcos-x86) [inline coding:1]: What is this file?

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**